rosetta-source/src/main/resources/result-json-files/fis/isla1.json

```diff
@@ -28,10 +28,6 @@
         "tradableProduct" : {
           "product" : {
             "contractualProduct" : {
-              "productTaxonomy" : [ {
-                "source" : "ISDA",
-                "productQualifier" : "SecurityLendingAgreement"
-              } ],
```

Comment:
These ones should continue to qualify (but as "SecuritiesFinance") once you rework the FIS Mapper.